Page2.html

```diff
@@ -0,0 +1,247 @@
+<!DOCTYPE html>
```

Comment:
Would recommend titling your pages by what they are, e.g.: "checkout.html" or "blog.html". Useful when others are viewing your code to know which files are which!